### PR TITLE
chore: restructuring of locale files and ko fr pt es locale update

### PR DESCRIPTION
**Pull request by LPX55**
## Description

- Translation locale files slightly restructured to play nicer with the temp CL platform
- ~~Borrowed `nd` locale code as an empty store so translators can visually see the context of what they are translating... sorry North Ndebele speakers~~
- Updated KO locale, rest to come in coming days.
- Edit: ES, PT, FR have been added to the updated list

## Notice

Before submitting a pull request, please make sure you have answered the following:

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

### CODE REVIEW

File: sample.env [hunk 1]
```diff
@@ -1,25 +0,0 @@
...
-REACT_APP_FEATURE_COSMOS_INVESTOR=false
-REACT_APP_FEATURE_PLUGIN_BITCOIN=false
-REACT_APP_FEATURE_PLUGIN_COSMOS=false
-REACT_APP_FEATURE_GEM_RAMP=false
-
-REACT_APP_GEM_COINIFY_SUPPORTED_COINS=https://api.gem.co/institutions/coinify/supported_currencies
-REACT_APP_GEM_WYRE_SUPPORTED_COINS=https://api.gem.co/institutions/wyre/supported_currencies
-REACT_APP_GEM_ASSET_LOGO=https://gem-widgets-assets.s3-us-west-2.amazonaws.com/currencies/crypto/
-REACT_APP_GEM_ENV=production
-REACT_APP_GEM_API_KEY=17baca45f492df6b8974d4f832a659c92644acafea3284cfb0d52550dd3bde1a
```
Comment (theoboldfrazier) [inline sample.env:25]: We don't want to delete this file.

**Review by theoboldfrazier — DISMISSED** [review]
@LPX55 This looks good. Please add back the sample.env file and merge in the latest develop and I'll approve

File: sample.env (hunk 1 above)
Comment (LPX55) [reply sample.env]: Whoops -- sample .env added back in.

File: src/assets/translations/index.ts [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
-import es from './messages-es.json'
-import fr from './messages-fr.json'
-import id from './messages-id.json'
-import ko from './messages-ko.json'
-import pt from './messages-pt.json'
-import ru from './messages-ru.json'
-import zh from './messages-zh.json'
+/* eslint-disable simple-import-sort/imports */
+import en from './main.json'
+import nd from './dev/main.json'
```
Comment (0xdef1cafe) [inline src/assets/translations/index.ts:3]: ?

File: src/assets/translations/index.ts [hunk 3]
```diff
@@ -1,10 +1,12 @@
-import en from './messages-en.json'
-import es from './messages-es.json'
-import fr from './messages-fr.json'
-import id from './messages-id.json'
-import ko from './messages-ko.json'
-import pt from './messages-pt.json'
-import ru from './messages-ru.json'
-import zh from './messages-zh.json'
+/* eslint-disable simple-import-sort/imports */
```
Comment (0xdef1cafe) [inline src/assets/translations/index.ts:1]:
```suggestion
```

File: sample.env [hunk 4]
```diff
@@ -22,4 +22,4 @@ REACT_APP_GEM_COINIFY_SUPPORTED_COINS=https://api.gem.co/institutions/coinify/su
 REACT_APP_GEM_WYRE_SUPPORTED_COINS=https://api.gem.co/institutions/wyre/supported_currencies
 REACT_APP_GEM_ASSET_LOGO=https://gem-widgets-assets.s3-us-west-2.amazonaws.com/currencies/crypto/
 REACT_APP_GEM_ENV=production
-REACT_APP_GEM_API_KEY=17baca45f492df6b8974d4f832a659c92644acafea3284cfb0d52550dd3bde1a
+REACT_APP_GEM_API_KEY=bb4164a72246dae1e03010d664d6cdae4e19b2554de02e3bf6c3cd30aa7e359e
```
Comment (0xdef1cafe) [inline sample.env:25]: this should not be changing in this PR

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]
see comments

File: sample.env (hunk 4 above)
Comment (LPX55) [reply sample.env]: Pulled this off the newest commit, reverting back to previous

File: src/assets/translations/index.ts (hunk 2 above)
Comment (LPX55) [reply src/assets/translations/index.ts]: eslint was giving errors

**Review by 0xApotheosis — CHANGES_REQUESTED** [review]

**Comment by 0xApotheosis** [comment]
Hey @LPX55,  could you also please update the PR description to accurately reflect the changes made?

E.g. the description states that only `KO` was updated, but we've touched `FR`, `ID`, `KO`, `PT`, `RU`, and `ZH`.

Better for version history when this gets merged 😄

**Comment by LPX55** [comment]
> Hey @LPX55, could you also please update the PR description to accurately reflect the changes made?
> 
> E.g. the description states that only `KO` was updated, but we've touched `FR`, `ID`, `KO`, `PT`, `RU`, and `ZH`.
> 
> Better for version history when this gets merged 😄

Alrighty, I think I got everything.

**Review by 0xApotheosis — APPROVED** [review]
There are some merge conflicts to resolve first, but otherwise looks good.

**Comment by LPX55** [comment]
> @LPX55 if you can rebase this off develop i'll stamp and get this in. apologies for the delay

rebased

**Comment by 0xdef1cafe** [comment]
> > @LPX55 if you can rebase this off develop i'll stamp and get this in. apologies for the delay
> 
> rebased.
> 
> Not sure if I did it correctly, lmk if I screwed up somehow. Sorry about the late response, been battling COVID.

ugh, sorry about the covid ser.

it looks like there is still conflicts - you can also just merge `develop` into this this branch if that's easier than rebasing for you.

**Comment by LPX55** [comment]
> > > @LPX55 if you can rebase this off develop i'll stamp and get this in. apologies for the delay
> > 
> > 
> > rebased.
> > Not sure if I did it correctly, lmk if I screwed up somehow. Sorry about the late response, been battling COVID.
> 
> ugh, sorry about the covid ser.
> 
> it looks like there is still conflicts - you can also just merge `develop` into this this branch if that's easier than rebasing for you.

Hopefully that did the trick -- not seeing any conflict errors on my end *fingers crossed*

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**